.env.local

```diff
@@ -0,0 +1,2 @@
+GOOGLE_MAPS_API_KEY=AIzaSyAn64kJ4rPULK-H4XnyGc_nJ_BxdHK-VAM
```

Comment:
http://funnyjunk.com/Internet+socializing/funny-pictures/4962602/

Reply:
Ah, I was wondering where that went.